### PR TITLE
changed a lot layouts

### PR DESCRIPTION
**Pull request by nishi-ai**
Screenshot 2022-01-15 at 12 57 41
Screenshot 2022-01-15 at 12 57 51
Screenshot 2022-01-15 at 12 58 13
Screenshot 2022-01-15 at 12 58 22

### CODE REVIEW

File: src/components/AddPlayerForm.js [hunk 1]
```diff
@@ -54,13 +61,13 @@ function AddPlayerForm(props) {
     };
 
         return (
-            <Form noValidate onSubmit={submitHandler} validated={enteredNameIsValid && enteredEmailIsValid}>
-                <Form.Group className="mb-3">
-                    <Form.Label htmlFor='playername'>Name</Form.Label>
+            <Form noValidate onSubmit={submitHandler} validated={enteredNameIsValid && enteredEmailIsValid} className='d-grid gap-2'>
```
Comment (benevbright) [inline src/components/AddPlayerForm.js:64]: Out of curiosity, what is `d-grid gap-2` class?

File: src/App.js [hunk 2]
```diff
@@ -22,14 +21,10 @@ class App extends Component {
...
+           <Title data={data.main} />
             <Route path='/' exact>
-            <GridContainer>
-              <h1 className="m-5">{title}</h1>
-                <Row className="justify-content-md-center">
-                  <Col sm><DisplayEventInfo /></Col>
-                  <Col sm><DisplayBasicInfo data={data.main} /></Col>
-                </Row> 
-            </GridContainer>
+            <DisplayBasicInfo data={data.main} />
```
Comment (benevbright) [inline src/App.js:26]: Let's add some indents.

**Review by benevbright — APPROVED** [review]
Amazing~!!! Super impressive work. 
And also the code looks very clean 🙌🙌🙌

File: src/components/AddPlayerForm.js (hunk 1 above)
Comment (nishi-ai) [reply src/components/AddPlayerForm.js:64]: To make the button size be adjusted with Form group, this makes full-width, according to [react-bootstrap](https://react-bootstrap.netlify.app/components/buttons/#buttons).

Comment (benevbright) [reply src/components/AddPlayerForm.js:64]: Cool. Thx 👍

**Review by benevbright — APPROVED** [review]

**Pull request merged**